multigeiger/multigeiger.ino

```diff
@@ -147,6 +147,14 @@ void publish(unsigned long current_ms, unsigned long current_counts, unsigned lo
     display_GMC(((int)accumulated_time / 1000), (int)(accumulated_Dose_Rate * 1000), (int)(Count_Rate * 60),
                 (showDisplay && switches.display_on));
 
+    // Sound local alarm?
+    if (soundLocalAlarm && GMC_factor_uSvph > 0) {
+      if ((accumulated_Dose_Rate > localAlarmThreshold) || (Dose_Rate > accumulated_Dose_Rate * 3)) {
+        log(INFO, "Local alarm: current dose %.3f vs. accumulated dose %.3f µSv/h", Dose_Rate, accumulated_Dose_Rate);
+        alarm();
+      }
```

Comment:
guess it would be good to have 2 IFs, one for each term and also to log a specific message for each condition, like:

1. "Local alarm: accumulated dose %.3f µSv/h > alarm threshold"
2. "Local alarm: current dose %.3f > 3 * accumulated dose %.3f µSv/h"

Reply:
and guess the 3 also needs to be configurable?

Reply:
Done